src/seedu/addressbook/Main.java

```diff
@@ -79,14 +80,24 @@ private void exit() {
     /** Reads the user command and executes it, until the user issues the exit command.  */
     private void runCommandLoopUntilExitCommand() {
         Command command;
-        do {
-            String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
-
-        } while (!ExitCommand.isExit(command));
+        try {
+            do {
+                String userCommandText = ui.getUserCommand();
+                
+                if (!storage.fileExists()) {
+                    throw new FileNotFoundException("Can't find storage file.");
+                }
```

Comment:
This method may be too high-level to detect this problem. This method doesn't even know there is a file involved.
`StorageFile.save()` may be a better place to detect the problem.